slides/advanced-01-introduction.qmd

```diff
@@ -153,129 +132,320 @@ If you are using your own laptop instead of Posit Cloud:
 #| eval: false
 
 # Install the packages for the workshop
-pkgs <- 
-  c("bonsai", "Cubist", "doParallel", "earth", "embed", "finetune", 
-    "lightgbm", "lme4", "parallelly", "plumber", "probably", 
-    "ranger", "rpart", "rpart.plot", "rules", "splines2", "stacks", 
-    "text2vec", "textrecipes", "tidymodels", "vetiver")
+pkgs <- c("almanac", "betacal", "bonsai", "C50", "Cubist", "dimRed", 
+          "doParallel", "earth", "embed", "extrasteps", "finetune", "forested", 
+          "igraph", "irlba", "kknn", "lightgbm", "lme4", "mirai", "parallelly", 
+          "plumber", "probably", "ranger", "RANN", "rpart", "rpart.plot", 
+          "RSpectra", "rules", "splines2", "stacks", "text2vec", "textrecipes", 
+          "tidymodels", "uwot", "vetiver")
 
 install.packages(pkgs)
 ```
 
-Also, you should install the newest version of the dials package (version 1.3.0). To check this, you can run: 
+Also, you should make sure that you have installed the newest version of a few packages. To check this, you can run: 
 
 ```{r}
 #| label: check-dials
 #| eval: false
 #| echo: true
-rlang::check_installed("dials", version = "1.3.0")
+rlang::check_installed("tidymodels", version = "1.4.1")
+rlang::check_installed("embed", version = "	1.2.0")
 ```
 
-## Hotel Data `r hexes("tidymodels", "dplyr")`
+# Let's get started!  {background-color="#B00000FF"}
 
-We'll use [data on hotels](https://www.sciencedirect.com/science/article/pii/S2352340918315191) to predict the cost of a room. 
+## Load tidymodels `r hexes("tidymodels")`
 
-The [data](https://modeldata.tidymodels.org/reference/hotel_rates.html) are in the modeldata package. We'll sample down the data and refactor some columns: 
+We're here to learn more about how to use the more advanced bits of tidymodels for supervised learning. 
 
-:::: {.columns}
+<br>
 
-::: {.column width="40%"}
+. . .
+
+First, let's load the meta-package: 
 
 ```{r}
-#| label: tune-startup
+#| label: load-tm
 library(tidymodels)
```

Comment:
This prints no output (first note). I'm pretty sure it is not loaded beforehand.